6.0-alpine/run.sh

```diff
@@ -6,8 +6,7 @@ if [ "${1:0:1}" != '-' ]; then
   exec "$@"
 fi
 
-exec java -jar lib/sonar-application-$SONAR_VERSION.jar \
-  ${SONARQUBE_JAVA_ARGS} \
+exec java ${SONARQUBE_JAVA_ARGS} -jar lib/sonar-application-$SONAR_VERSION.jar \
```

Comment:
@pwagland my best guess (taking into account current time) to be verified - is that you can do it as:
```
docker run -p 9000:9000 -it sonarqube:5.6.1 -Dsonar.ce.javaOpts=-Xmx...
```
See https://github.com/SonarSource/sonarqube/blob/5.6.1/sonar-application/src/main/assembly/conf/sonar.properties#L137